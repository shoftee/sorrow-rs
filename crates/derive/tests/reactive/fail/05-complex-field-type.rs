use sorrow_derive::Reactive;

#[derive(Reactive)]
pub struct ComplexFieldTypes {
    complex: *const i32,
}

fn main() {}
