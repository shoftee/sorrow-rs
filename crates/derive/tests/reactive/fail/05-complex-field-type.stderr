error: derive(Reactive) only supports basic types.
 --> tests/reactive/fail/05-complex-field-type.rs:5:14
  |
5 |     complex: *const i32,
  |              ^^^^^^^^^^
